.github/workflows/pr-alert.yml

```diff
@@ -1,14 +1,20 @@
 name: Circuit PR Alerting
 
-on: pull_request
+on:
+  pull_request:
+    paths:
+      - '**/*.rs'
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+      - .github/workflows/circuit.yml
 
 jobs:
   alert:
     runs-on: ubuntu-20.04
-    if: ! github.event.pull_request.draft
+    if: ${{ !github.event.pull_request.draft }}
     steps:
       - name: Await checks
-        uses: fountainhead/action-wait-for-check@v1
+        uses: fountainhead/action-wait-for-check@v1.0.0
```

Comment:
leave for any? `@*`

Reply:
yea, somehow actions was complaining it couldn't find the version, just stripped it now